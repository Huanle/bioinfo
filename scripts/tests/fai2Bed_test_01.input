chr1	58871917	6	50	51
chr10	45574255	60049369	50	51
chr11	45107271	106535117	50	51
chr12	49229541	152544541	50	51
chr13	51780250	202758680	50	51
chr14	51944548	255574542	50	51
chr15	47771147	308557988	50	51
chr16	55381981	357284565	50	51
chr17	53345113	413774193	50	51
chr18	51008593	468186216	50	51
chr19	48790377	520214988	50	51
chr2	59543403	569981179	50	51
chr20	55370968	630715458	50	51
chr21	45895719	687193853	50	51
chr22	39226288	734007494	50	51
chr23	46272358	774018315	50	51
chr24	42251103	821216128	50	51
chr25	36898761	864312261	50	51
chr3	62385949	901949004	50	51
chr4	76625712	965582678	50	51
chr5	71715914	1043740911	50	51
chr6	60272633	1116891150	50	51
chr7	74082188	1178369242	50	51
chr8	54191831	1253933080	50	51
chr9	56892771	1309208754	50	51
chrUn_KN149679v1	35552	1367239399	50	51
chrUn_KN149680v1	38724	1367275681	50	51
chrUn_KN149681v1	107758	1367315198	50	51
chrUn_KN149682v1	137468	1367425130	50	51
chrUn_KN149683v1	152860	1367565366	50	51
