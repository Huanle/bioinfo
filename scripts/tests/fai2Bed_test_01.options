-i fai2Bed_test_01.input -q
