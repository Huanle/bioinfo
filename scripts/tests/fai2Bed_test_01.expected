chr1	0	58871917
chr10	0	45574255
chr11	0	45107271
chr12	0	49229541
chr13	0	51780250
chr14	0	51944548
chr15	0	47771147
chr16	0	55381981
chr17	0	53345113
chr18	0	51008593
chr19	0	48790377
chr2	0	59543403
chr20	0	55370968
chr21	0	45895719
chr22	0	39226288
chr23	0	46272358
chr24	0	42251103
chr25	0	36898761
chr3	0	62385949
chr4	0	76625712
chr5	0	71715914
chr6	0	60272633
chr7	0	74082188
chr8	0	54191831
chr9	0	56892771
chrUn_KN149679v1	0	35552
chrUn_KN149680v1	0	38724
chrUn_KN149681v1	0	107758
chrUn_KN149682v1	0	137468
chrUn_KN149683v1	0	152860
